main.c

```diff
@@ -218,6 +218,8 @@ static const struct fuse_opt ovl_opts[] = {
    offsetof (struct ovl_data, plugins), 0},
   {"xattr_permissions=%d",
    offsetof (struct ovl_data, xattr_permissions), 0},
+  {"squash_to_root",
```

Comment:
should this be `squash_to_root=%d` or would it work as a boolean if we don't specify the type?

Reply:
Seems to work as a Boolean. I was trying to follow the docs (https://libfuse.github.io/doxygen/structfuse__opt.html).